quickstarts/gcp/apigee-api/config.yml

```diff
@@ -0,0 +1,30 @@
+# Name of the quickstart (required)
+name: apigee-api
+title: Apigee API Distributed Tracing
+description: |
+  Apigee API JavaScript Resources to create or propagate W3C Trace Context through Apigee Flows and on to the Apigee Target. Apigee Flow Spans are reported back to New Relic's Trace API. Sample Policies are included to demonstrate use.
+
+summary: |
+  Monitor Apigee API Flows with New Relic's Trace API
+
+logo: logo.svg
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+website: https://cloud.google.com/apigee
+
+keywords:
+  - nrlabs
+  - nrlabs-data
```

Comment:
```suggestion
```

Reply:
suggesting we use a single keyword for nrlabs.

Reply:
@jpvajda  All of our (New Relic Labs) quickstarts use both. I believe it will break some automated reports if we drop one.

Reply:
oh ok, that's cool we can leave it. thanks!